### PR TITLE
update apps.json for portal quickstart

### PR DESCRIPTION
**Pull request by hamiltonha**
## Purpose

*  Update for the portal quickstart. AppID cannot be "Enter_the_app_id_here" in the apps.json file, has to be set to the resource app id. Because of the special logic for generating reply URLS for android, there is no need for the replyUrlsWithType section - deleted. 

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by rpdome** [comment]
@hamiltonha what is this change for?

**Comment by hamiltonha** [comment]
@rpdome ok if I merge?

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -9,19 +9,13 @@
...
-        }
-      ],
       "requiredResourceAccess": [
         {
           "x-ms-resourceAppName": "Microsoft Graph",
           "resourceAppId": "00000003-0000-0000-c000-000000000000",
           "resourceAccess": [
             {
-              "id": "Enter_the_Application_Id_here",
+              "id": "e1fe6dd8-ba31-4d61-89e7-88639da4683d",
```
Comment (rpdome) [inline AppCreationScripts/apps.json:18]: Where are you getting this id from?

Comment (TrishalaPugazhendhi) [reply AppCreationScripts/apps.json:18]: In the apps.json file, the resource App Id was initially set to "e1fe6dd8-ba31-4d61-89e7-88639da4683d" which is the correct resourceAppId for User.Read. The resourceAppID is set to "Enter_the_application_Here" here. The application AppId shouldn't be the resouce App ID in requiredResourceAccess. If the required resource is User.Read of MSGraph then the resource App Id should be set to "e1fe6dd8-ba31-4d61-89e7-88639da4683d".

Comment (rpdome) [reply AppCreationScripts/apps.json:18]: fyi: https://marketplace.visualstudio.com/items?itemName=stephane-eyskens.aadv1appprovisioning

e1fe6dd8-ba31-4d61-89e7-88639da4683d is associated to MSGraph's User.Read

**Pull request merged**